tests/librarything-search-index-conflicts.unit.php

```
@@ -0,0 +1,154 @@
+$db->set_charset('utf8mb4');
+
+$hasSearchIndex = $db->query("SHOW COLUMNS FROM libri LIKE 'search_index'");
+if (!$hasSearchIndex || $hasSearchIndex->num_rows === 0) {
+    echo "SKIP: libri.search_index not present\n";
+    exit(0);
+}
+
+$tag = 'ZZ_LT_SIC_' . bin2hex(random_bytes(4));
+$isbn = '979' . random_int(1000000000, 9999999999);
+
+function ltSicCleanup(mysqli $db, string $tag): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)